ophyd/tests/mca_ioc.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+from caproto.server import (pvproperty, PVGroup, ioc_arg_parser, run, SubGroup,
+                            get_pv_pair_wrapper)
+from caproto import ChannelType
+
+pvproperty_with_rbv = get_pv_pair_wrapper(setpoint_suffix='',
+                                          readback_suffix='_RBV')
+unknown = int
```

Comment:
This looks odd. Is it just a quick way to set up defaults?

Reply:
Good catch! Yes, it's what `caproto.server.conversion.ophyd_device_to_caproto_ioc` gives for `dtype` when you only have a `Device` class and not an instance which can talk to a real IOC.

The changes from `unknown` to some dtype were the minimum changes necessary to mock the IOC for ophyd's usage. If these PVs become important later, we will have to revisit the caproto IOC.